skeleton/extension/phpstan.neon

```diff
@@ -1,9 +1,13 @@
 includes:
     - vendor/phpstan/phpstan-phpunit/extension.neon
     - vendor/phpstan/phpstan-symfony/extension.neon
+    - vendor/phpstan/phpstan-strict-rules/rules.neon
+    - vendor/ibexa/phpstan/extension.neon
 
 parameters:
-    level: 8
+    level: 10
```

Comment:
Could you please revert this change? I would like to first better understand impact on our code base, before this will become standard in our packages.